compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala

```diff
@@ -53,6 +53,11 @@ final class Conversions(
       .addAllScenarioSteps(steps.asJava)
       .setReturnValue(convertSValue(svalue))
       .setFinalTime(ledger.currentTime.micros)
+      .addAllActiveContracts(
+        ledger.ledgerData.activeContracts.toList
```

Comment:
```suggestion
        ledger.ledgerData.activeContracts.view
```

Reply:
Its possible to just drop the `toList` altogether